Sources/BottomSheetView.swift

```diff
@@ -120,17 +120,24 @@ public final class BottomSheetView: UIView {
 
         springAnimator.addCompletion { didComplete in completion?(didComplete) }
 
-        NSLayoutConstraint.activate([
+        var constraints: [NSLayoutConstraint] = [
             topConstraint,
             bottomAnchor.constraint(greaterThanOrEqualTo: superview.bottomAnchor),
             leadingAnchor.constraint(equalTo: superview.leadingAnchor),
             trailingAnchor.constraint(equalTo: superview.trailingAnchor)
-        ])
+        ]
+
+        updateTargetOffsets()
+
+        if let maxOffset = targetOffsets.max() {
+            let contentViewHeight = superview.frame.size.height - maxOffset - BottomSheetCalculator.handleHeight
```

Comment:
Do we want to subtract `handleHeight` here?

Reply:
Yes, because `contentViewHeight = totalBottomSheet - handleHeight` 😄

Reply:
And all target offsets include handle height.